grid/social-media/followers/profile.supr

```diff
@@ -51,6 +51,32 @@ usecase GetFollowers safe {
       """
       imageUrl string
     } ]
+    """
+    Rate limit
+    Rate limit details.
+    """
+    rateLimit {
+      """
+      Total requests available
+      Total requests available in the time window.
+      """
+      totalRequestsAvailable number
+      """
+      Remaining requests available
+      Remaining requests available in the time window.
+      """
+      remainingRequestsAvailable number
+      """
+      Remaining requests available in percents
+      Remaining requests available in the time window in percents.
+      """
+      remainingRequestsAvailableInPercent number
```

Comment:
It's not shorter but maybe more readable?

```suggestion
      remainingRequestsAvailablePercentage number
```

Reply:
changed